packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -0,0 +1,669 @@
+{{page-title "Modal Component"}}
+
+<h2 class="dummy-h2">Modal</h2>
+
+{{! ADD YOUR CONTENT BELOW }}
+{{! Please follow as much as possible what it's already done in other components. }}
+{{! You can start by copy&pasting it, if you want, and then adapt it according to your needs. }}
+{{! Once done, please remove these comments (they're created by the generator). }}
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Modals are used as an interstitial to request information or feedback from the user,
```

Comment:
[Suggestion, non-blocker] I don't think I've seen the word `interstitial` used outside of a medical context. Perhaps a link to an acceptable definition on the web would be useful, especially for users who are unfamiliar with the word or have English has a non-primary language.

Reply:
Yeah, I can see how this may sound like that (many Latin words often have these scientific connotations); the intention, I guess, was to express the fact that modals are usually placed between two states – as interacting with the modal changes the state of what invoked the modal. Luckily for me, Jory provided new wording for this section which does not include this one.